central/telemetry/service/service_impl.go

```diff
@@ -51,12 +51,14 @@ func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName strin
 	return ctx, authorizer.Authorized(ctx, fullMethodName)
 }
 
+// GetTelemetryConfiguration is deprecated.
```

Comment:
```suggestion
// GetTelemetryConfiguration do ...
// Deprecated: <reason>
```
https://go.dev/blog/godoc#:~:text=that%20begins%20with-,%E2%80%9CDeprecated%3A%E2%80%9D,-followed%20by%20some